src/copyabletreeview.h

```diff
@@ -0,0 +1,19 @@
+/*
+    SPDX-FileCopyrightText: Lieven Hey <lieven.hey@kdab.com>
+    SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+
+    SPDX-License-Identifier: GPL-2.0-or-later
+*/
+
+#pragma once
+
+#include <QTreeView>
+
+class CopyableTreeView : public QTreeView
+{
+    Q_OBJECT public : CopyableTreeView(QWidget* parent = nullptr);
```

Comment:
bad formatting, missing `explicit`

Reply:
oh wow, apparently this is valid clang formatting